manifests/database/ssl_configuration.pp

```diff
@@ -6,7 +6,7 @@
   $database_username           = $puppetdb::params::database_username,
   $read_database_username      = $puppetdb::params::read_database_username,
   $read_database_host          = $puppetdb::params::read_database_host,
-  $puppetdb_server             = $puppetdb::params::puppetdb_server,
+  Variant[String,Array[String, 1]] $puppetdb_server = $puppetdb::params::puppetdb_server,
```

Comment:
I think that's the least we should do. would even be better to enforce IP/FQDN, but I don't know out of my head what kind of values are allowed here.
```suggestion
  Variant[String[1],Array[String[1], 1]] $puppetdb_server = $puppetdb::params::puppetdb_server,
```